sdcm/sct_config.py

```diff
@@ -1606,17 +1599,13 @@ def __init__(self):
         add_severity_limit_rules(self.get("max_events_severities"))
         print_critical_events()
 
-        # 5) assume multi dc by n_db_nodes set size
-        if 'aws' in cluster_backend:
```

Comment:
I thought this is a check

Now I see this is some kind of old hack to get data out of region_data even that we already did it...